sql-database/failover-groups/add-single-db-to-failover-group-az-ps.ps1

```diff
@@ -0,0 +1,130 @@
+# Set variables for your server and database
+$subscriptionId = '<SubscriptionID>'
+$resourceGroupName = "myResourceGroup-$(Get-Random)"
+$location = "West US 2"
+$adminLogin = "azureuser"
+$password = "PASSWORD"+(New-Guid).Guid
```

Comment:
__Recommended__: Avoid the `PASSWORD` prefix and just use the GUID.

Reply:
@sptreamer the complexity requirements of Azure SQL Database require upper and lower case characters. the New-Guid command does not return any upper case characters